docs/source/notebooks/gp-optimizer-4d.ipynb

```diff
@@ -102,21 +96,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dir(w9)"
+    "gpo.plot_fitness()"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "1017c3f1-3ba4-45ec-bc84-d176af999990",
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "gp",
```

Comment:
Just a note -- this may break the local execution in other differently named environments. But let's focus on making sure it will be rendered by sphinx.